inc/options/browsercache.php

```diff
@@ -431,36 +466,42 @@
+					<a class="w3tc-control-after" target="_blank" href="<?php echo esc_url( 'https://www.boldgrid.com/support/w3-total-cache/configuring-browser-caching-in-w3-total-cache#set-entity-tag-etag-cssjs' ); ?>"><?php esc_html_e( 'Learn more', 'w3-total-cache' ); ?><span class="dashicons dashicons-external"></span></a>
 					<p class="description"><?php esc_html_e( 'Set the ETag header to encourage browser caching of files.', 'w3-total-cache' ); ?></p>
 				</th>
 			</tr>
 			<tr>
 				<th colspan="2">
 					<?php $this->checkbox( 'browsercache.cssjs.w3tc' ); ?> <?php Util_Ui::e_config_label( 'browsercache.cssjs.w3tc' ); ?></label>
+					<a class="w3tc-control-after" target="_blank" href="<?php echo esc_url( 'https://www.boldgrid.com/support/w3-total-cache/configuring-browser-caching-in-w3-total-cache#set-w3-total-cache-header-cssjs' ); ?>"><?php esc_html_e( 'Learn more', 'w3-total-cache' ); ?><span class="dashicons dashicons-external"></span></a>
 					<p class="description"><?php esc_html_e( 'Set this header to assist in identifying optimized files.', 'w3-total-cache' ); ?></p>
 				</th>
 			</tr>
 			<tr>
 				<th colspan="2">
 					<?php $this->checkbox( 'browsercache.cssjs.compression' ); ?> <?php Util_Ui::e_config_label( 'browsercache.cssjs.compression' ); ?>  </label>
+					<a class="w3tc-control-after" target="_blank" href="<?php echo esc_url( 'https://www.boldgrid.com/support/w3-total-cache/configuring-browser-caching-in-w3-total-cache#enable-http-gzip-compression-cssjs' ); ?>"><?php esc_html_e( 'Learn more', 'w3-total-cache' ); ?><span class="dashicons dashicons-external"></span></a>
 					<p class="description"><?php esc_html_e( 'Reduce the download time for text-based files.', 'w3-total-cache' ); ?></p>
 				</th>
 			</tr>
 			<tr>
 				<th colspan="2">
 					<?php $this->checkbox( 'browsercache.cssjs.brotli', ! function_exists( 'brotli_compress' ) ); ?> <?php Util_Ui::e_config_label( 'browsercache.cssjs.brotli' ); ?>  </label>
+					<a class="w3tc-control-after" target="_blank" href="<?php echo esc_url( 'https://www.boldgrid.com/support/w3-total-cache/configuring-browser-caching-in-w3-total-cache#enable-http-brotli-compression-cssjs' ); ?>"><?php esc_html_e( 'Learn more', 'w3-total-cache' ); ?><span class="dashicons dashicons-external"></span></a>
 					<p class="description"><?php esc_html_e( 'Reduce the download time for text-based files.', 'w3-total-cache' ); ?></p>
 				</th>
 			</tr>
 			<tr>
 				<th colspan="2">
 					<?php $this->checkbox( 'browsercache.cssjs.replace' ); ?> <?php Util_Ui::e_config_label( 'browsercache.cssjs.replace' ); ?></label>
+					<a class="w3tc-control-after" target="_blank" href="<?php echo esc_url( 'https://www.boldgrid.com/support/w3-total-cache/configuring-browser-caching-in-w3-total-cache#remove-query-strings-from-static-resources-cssjs' ); ?>"><?php esc_html_e( 'Learn more', 'w3-total-cache' ); ?><span class="dashicons dashicons-external"></span></a>
```

Comment:
Incorrect link for item "Prevent caching of objects after settings change".  The item is missing from the documentation.

Reply:
That link works for me? I know the documentation page anchor scrolling isn't perfect so my guess is it "kickuped" for you. Please test again